VerilogCodingStyle.md

```diff
@@ -2385,9 +2407,10 @@ a check for `X` on the 4-state net before resolving to a 2-state variable.
 ***Use logical constructs for logical comparisons, bit-wise for data.***
 
 Logical constructs (`!`, `||`, `&&`, `==`, `!=`) should be used for all
-constructs that are evaluating logic (true or false) values, such as
-if clauses and ternary assignments.  Use bit-wise constructs (`~`, `|`,
-`&`, `^`) for all data constructs, even if scalar.
+constructs that are evaluating control path logic. This includes boolean
+logic (true or false) values, such as if clauses and ternary assignments.
```

Comment:
I have a recommended rewrite for this section in #21, let's do that in a
separate PR please. I can initiate that one. I feel this wording goes
all the way to the other side of the current guidance, while #21 was
recommending allowing both styles in different contexts.